parsing/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -2851,4 +2851,125 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
     sql(sql).fails(expected);
   }
 
+  @Test void testClob() {
+    final String sql = "create table foo (bar clob)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` CLOB)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testCharacterLargeObject() {
+    final String sql = "create table foo (bar character large object)";
```

Comment:
is setting a CLOB value + using `character large object` a valid use case?

Reply:
Yeah, I could add a test case for that too